GHAS-on-GHES-feature-matrix.md

```diff
@@ -178,6 +181,8 @@ Security overview provides high-level summaries of the security status of an org
 |------------------------------------------------------------|-----|-----|-----|-----|-----|-----|----|----|----|
 |[Security Managers Role](https://docs.github.com/en/enterprise-server/organizations/managing-peoples-access-to-your-organization-with-roles/managing-security-managers-in-your-organization)|✅|✅|✅|✅|✅|✅|✅|✅|✅|
 |[Manage Security Managers role via the API](https://docs.github.com/en/enterprise-server/rest/orgs/security-managers?apiVersion=2022-11-28)||||✅|✅|✅|✅|✅|✅|
+|[Licensing for committers only after the migration date](https://docs.github.com/en/enterprise-server@3.12/billing/managing-billing-for-github-advanced-security/about-billing-for-github-advanced-security) |||||||||✅|
```

Comment:
Interesting - in the release notes for 3.12 (https://docs.github.com/en/enterprise-server@3.12/admin/release-notes)


> For repositories migrated to GitHub Enterprise Server from other platforms, to calculate active committers for GitHub Advanced Security license usage, GitHub only considers commits made after the migration date. Previously, historic commits were included in the calculation, and users needed to intervene manually to avoid consuming licenses unnecessarily. For more information, see "[About billing for GitHub Advanced Security](https://docs.github.com/en/enterprise-server@3.12/billing/managing-billing-for-github-advanced-security/about-billing-for-github-advanced-security)."

Reply:
pretty sure this is a somewhat recent fix i have seen fly by in GHEC ( last 6 months or so)